dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -315,9 +321,148 @@ public void saveScheme(final WorkflowScheme scheme, final User user) throws DotD
 
 	}
 
+
+	public void deleteScheme(final WorkflowScheme scheme, final User user)
+			throws DotDataException, DotSecurityException, AlreadyExistException {
+

```

Comment:
missing the validation for the isUserAllowToModifiedWorkflow

Reply:
done